protos/ensembl/production/metadata/ensembl_metadata.proto

```diff
@@ -56,6 +59,9 @@ message Genome {
   bool is_current = 5;
   Assembly assembly = 6;
   Taxon taxon = 7;
```

Comment:
Note that I haven't updated the `Taxon` message to match the `taxonomy` field in https://www.ebi.ac.uk/panda/jira/secure/attachment/42618/genome_uuid_list.json.  I couldn't see where to get `species_taxonomy_id` and I'm not sure whether we should keep `taxonomy` or use `taxon` instead.

Reply:
All fine.